readme.md

```diff
@@ -0,0 +1,29 @@
+# sphinxext-opengraph
+Sphinx extension to generate OpenGraph metadata (https://ogp.me/)
+
+## Options
+These values are placed in the conf.py of your sphinx project.
+
+### Site URL
+- *Formal Name:* `ogp_site_url`
```

Comment:
Could you reformat this to be something that shows an example configuration. Also Formal Name is confusing, what is Formal Name? You could just do `ogp_description_length: 200`